### PR TITLE
Update landing page

### PR DESCRIPTION
**Pull request by mateuszc-kainos**
## Description
Update landing page

## Jira Ticket

https://companieshouse.atlassian.net/browse/BI-6215

## Coding Standards

Code Style Guide: https://github.com/companieshouse/dissolution-web/wiki/Code-Style-Guide

## Checklist

- [ ] 3 Amigos
- [x] Acceptance Criteria met
- [x] Branch named {feature|hotfix|task}/{S4-*}
- [x] Commit messages are meaningful
- [x] Manually tested
- [x] All unit tests passing
- [x] WAVE accessibility testing tool ran on new or updated pages
- [ ] All UI Tests Passing
- [ ] Pulled latest master into feature branch
- [ ] Sonar Analysis
- [ ] New Docker environment variables are consistent with those on the Rebel1 environment
- [ ] If your pull request depends on any other, please link them in the description

## Screenshots of new or updated views
Screenshot 2020-11-26 at 09 21 15

### CODE REVIEW

**Comment by mateuszc-kainos** [comment]
The layout and content has been accepted by Jayti.

**Comment by mateuszc-kainos** [comment]
image

Toggle off

File: src/middleware/nunjucksLoader.middleware.ts [hunk 1]
```diff
@@ -17,7 +17,8 @@ export default class NunjucksLoader {
 
   public constructor(
     @inject(TYPES.CDN_HOST) private CDN_HOST: string,
-    @inject(TYPES.PIWIK_CONFIG) private PIWIK_CONFIG: PiwikConfig
+    @inject(TYPES.PIWIK_CONFIG) private PIWIK_CONFIG: PiwikConfig,
+    @inject(TYPES.PAY_BY_ACCOUNT_FEATURE_ENABLED) private PAY_BY_ACCOUNT_FEATURE_ENABLED: number,
```
Comment (matthewdoreilly) [inline src/middleware/nunjucksLoader.middleware.ts:21]: Remove trailing comma

File: src/middleware/nunjucksLoader.middleware.ts [hunk 2]
```diff
@@ -57,5 +58,7 @@ export default class NunjucksLoader {
     app.locals.pageTitleSuffix = PAGE_TITLE_SUFFIX
 
     app.locals.nonce = nonce
+
+    app.locals.payByAccountEnabled = this.PAY_BY_ACCOUNT_FEATURE_ENABLED
```
Comment (matthewdoreilly) [inline src/middleware/nunjucksLoader.middleware.ts:62]: Being picky, but can we make the nunjucks variable `payByAccountFeatureEnabled` as well?

**Review by matthewdoreilly — APPROVED** [review]
Just have a few minor comments

File: src/middleware/nunjucksLoader.middleware.ts (hunk 2 above)
Comment (mateuszc-kainos) [reply src/middleware/nunjucksLoader.middleware.ts]: Done

File: src/middleware/nunjucksLoader.middleware.ts (hunk 1 above)
Comment (mateuszc-kainos) [reply src/middleware/nunjucksLoader.middleware.ts]: Done

**Pull request merged**